lib/grpcServer/handlers/core/getEstimatedTransactionFeeHandlerFactory.js

```diff
@@ -0,0 +1,50 @@
+const {
+  GetEstimatedTransactionFeeResponse,
+} = require('@dashevo/dapi-grpc');
+
+const {
+  server: {
+    error: {
+      InternalGrpcError,
+    },
+  },
+} = require('@dashevo/grpc-common');
+
+/**
+ * @param {InsightAPI} insightAPI
+ * @returns {getEstimatedTransactionFeeHandler}
+ */
+function getEstimatedTransactionFeeHandlerFactory(insightAPI) {
+  const BLOCKS_BY_DEFAULT = 3;
+
+  /**
+   * @typedef getEstimatedTransactionFeeHandler
+   * @param {Object} call
+   * @returns {Promise<GetEstimatedTransactionFeeResponse>}
+   */
+  async function getEstimatedTransactionFeeHandler(call) {
+    const { request } = call;
+
+    let blocks = request.getBlocks();
+    if (blocks === null) {
```

Comment:
Maybe `==` instead of `===` ?

Reply:
So we can catch `undefined` as well. Or it can't happen with grpc?

Reply:
I would stay with strict typE comparison and just check undefined as well if needed. It’s one of the most important vest practices.

Reply:
We are all wrong, because for numeric types, the default value is zero in protobuf3. [doc link](https://developers.google.com/protocol-buffers/docs/proto3#default)